index.html

```diff
@@ -46,6 +46,204 @@ <h1>I'm Naan.
             </div>
         </headline>
         <!-- END OF HEADLINE SECTION-->
+
+        <!--Works Section-->
+        <!--Work 1-->
+        <div class="primary_text">
+            <div class="works">
+                <div class="card_works">
+                    <div class="snapshot_portfolio">
+                    </div>
+                    <div class="left_block underline">
+                        <div class ="primary_text">
+                            <div class="project_title">Tonic</div>
+                            <div class="sub_header">
+                                <div class="client">
+                                    <div class="client_text">Canopy</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <div class="text">Back End Dev</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
```

Comment:
When you have such a list of something, like navbar links, or tags like this div, then you should use `<ul>` with a list of `<li>`s in it. The purpose is the semantic logic, that browser needs to read and try to understand what these elements are alike and are in the form of a list. So keep the semantics in mind.
Also implement this in other work cards too.